forc-plugins/forc-tx/src/lib.rs

```diff
@@ -668,15 +659,19 @@ impl TryFrom<Script> for fuel_tx::Script {
             .into_iter()
             .map(|s| fuel_tx::Witness::from(s.as_bytes()))
             .collect();
+
+        let mut policies = Policies::default().with_maturity(script.maturity.maturity.into());
+        policies.set(PolicyType::GasPrice, script.gas.price);
         let script = fuel_tx::Transaction::script(
-            script.gas.price.unwrap_or_default(),
-            script
-                .gas
-                .limit
-                .unwrap_or(fuel_tx::ConsensusParameters::DEFAULT.max_gas_per_tx),
-            script.maturity.maturity.into(),
+            script.gas.script_gas_limit.unwrap_or(
+                fuel_tx::ConsensusParameters::default()
+                    .tx_params()
+                    .max_gas_per_tx
+                    / 4,
```

Comment:
Is this correct to divide by 4? If so, should there be another field we can use? Or at the very least a comment explaining what the logic here is? It's not clear to me from reading the code why this value is used. cc @xgreenx